python/cuxfilter/charts/deckgl/bindings/panel_deck.py

```diff
@@ -45,7 +45,7 @@ class PanelDeck(param.Parameterized):
 
     @property
     def valid_indices(self):
-        return self.indices.intersection(self.data.index)
+        return list(self.indices.intersection(self.data.index))
```

Comment:
I hope this conversion from device to host is an informed decision.

Reply:
Ah I can see why that's confusing, but `self.data` here is actually a pandas dataframe here, which is converted from a cudf Dataframe post aggregations, for direct rendering, so technically there is no device to host conversion.

Can see the ref on L34 same file, where data is of type param.DataFrame(), which only accepts pandas as of now

Reply:
ah, thanks for the clarification, Ajay!